proto/xudrpc.proto

```diff
@@ -203,6 +203,17 @@ service Xud {
     };
   }
 
+  /* Gets an order book depth chart where orders are grouped into "buckets"
+   * according to their price rounded to a given level of precision.
+   * shell: xucli orderbook [pair_id] [precision] */
+  rpc OrderBook(OrderBookRequest) returns (OrderBookResponse) {
+    option (google.api.http) = {
+      post: "/v1/orderbook"
```

Comment:
Would it make sense as a get request instead?

Reply:
I thought this too but the proto definition wouldn't accept `get` here since the request has a payload.